akka-docs/rst/java/http/routing-dsl/directives/basic-directives/extractStrictEntity.rst

```diff
@@ -0,0 +1,21 @@
+.. _-extractStrictEntity-java-:
+
+extractStrictEntity
+===================
+
+Description
+-----------
+
+Extracts the strict http entity as ``HttpEntity.Strict`` from the :class:`RequestContext`.
+
+A timeout parameter is given and if the stream isn't completed after the timeout, the directive will be failed.
+
+.. warning::
+
+  The directive will read the entire request entity into memory regardless of size and effectively disable streaming.
```

Comment:
Sounds good, maybe mention it can be configured globally with `akka.http.parsing.max-content-length` and add links to the `withSizeLimit` and `withoutSizeLimit` directives for route local overriding also?